src/test/java/core/basesyntax/service/impl/TransactionProcessorImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionProcessor;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TransactionProcessorImplTest {
+    private static TransactionProcessor transactionProcessor;
+
+    @BeforeClass
+    public static void setUp() {
+        transactionProcessor = new TransactionProcessorImpl();
+    }
+
+    @Test
+    public void process_validTransactions_ok() {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        transactions.add(new FruitTransaction(FruitTransaction.Operation.BALANCE, "banana", 20));
+        transactions.add(new FruitTransaction(FruitTransaction.Operation.PURCHASE, "banana", 6));
+        transactions.add(new FruitTransaction(FruitTransaction.Operation.RETURN, "banana", 3));
+        int expected = 17;
```

Comment:
```suggestion
        FruitTransaction balanceTransaction = new FruitTransaction(FruitTransaction.Operation.BALANCE, "banana", 20);
        FruitTransaction purchaseTransaction = new FruitTransaction(FruitTransaction.Operation.PURCHASE, "banana", 6);
        FruitTransaction returnTransaction = new FruitTransaction(FruitTransaction.Operation.RETURN, "banana", 3);
        transactions.add(balanceTransaction);
        transactions.add(purchaseTransaction);
        transactions.add(returnTransaction);
        int expected = balanceTransaction.getQuantity() - purchaseTransaction.getQuantity() + returnTransaction.getQuantity();
```
avoid hardcode where it's possible, look through other test methods and check if you can avoid hardcode

Reply:
✅ Thank you!)